modules/manage/partials/audit-logging.adoc

```diff
@@ -329,3 +365,11 @@ include::shared:partial$suggested-reading.adoc[]
 
 - xref:reference:topic-properties.adoc[]
 - xref:develop:config-topics.adoc[]
+endif::[]
+
+ifdef::env-cloud[]
```

Comment:
Also double check - is this page https://deploy-preview-1041--redpanda-docs-preview.netlify.app/redpanda-cloud/manage/cluster-maintenance/audit-logging supposed to display more than these two sections?

Reply:
confirmed content